File: u3u2/Assets/Scripts/app/net/c2s/CGPauseGuaJi.cs
using System;
using System.IO;
namespace app.net
{

/**
 * 暂停挂机
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGPauseGuaJi :BaseMessage
{
	
	
	public CGPauseGuaJi ()
	{
	}
	
	
	protected override void ReadImpl() 
	{
		return;
	}
	
	protected override void WriteImpl() 
	{

	}
	
	public override short GetMessageType()
	{
		return (short)MessageType.CG_PAUSE_GUA_JI;
	}
	
	public override string getEventType()
	{
		return "";
	}
	
	}
}

File: IdeaServer/zlj2019/game_server/src/com/imop/lj/gameserver/guaji/msg/CGPauseGuaJi.java
package com.imop.lj.gameserver.guaji.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.CGMessage;
import com.imop.lj.gameserver.guaji.handler.GuajiHandlerFactory;

/**
 * 暂停挂机
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGPauseGuaJi extends CGMessage{
	
	
	public CGPauseGuaJi (){
	}
	
	
	@Override
	protected boolean readImpl() {


		return true;
	}
	
	@Override
	protected boolean writeImpl() {

		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.CG_PAUSE_GUA_JI;
	}
	
	@Override
	public String getTypeName() {
		return "CG_PAUSE_GUA_JI";
	}


	@Override
	public void execute() {
		GuajiHandlerFactory.getHandler().handlePauseGuaJi(this.getSession().getPlayer(), this);
	}
}